plugin-server/tests/worker/ingestion/timestamps.test.ts

```diff
@@ -48,6 +56,20 @@ describe('parseEventTimestamp()', () => {
         expect(eventSecondsBeforeNow).toBeLessThan(610)
     })
 
+    it('ignores invalid sent_at', () => {
+        const rightNow = DateTime.utc()
+        const tomorrow = rightNow.plus({ days: 1, hours: 2 })
+
+        const event = {
+            timestamp: tomorrow.toISO(),
+            now: rightNow,
+            sent_at: 'invalid',
+        } as any as PluginEvent
+
+        const timestamp = parseEventTimestamp(event)
+        expect(timestamp).toEqual(tomorrow)
+    })
+
     it('captures sent_at with no timezones', () => {
         const rightNow = DateTime.utc()
```

Comment:
this would no longer be rightNow if we freeze time for everything, maybe make these be some specific times like `DateTime.fromISO('2020-01-01T12:00:05.200Z')` to make sure we don't hide problems now due to frozen time

Reply:
Thanks! Will address in the next PR.